package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.4.6",
   "description": "An implementation of the WebDriver BiDi protocol for Chromium implemented as a JavaScript layer translating between BiDi and CDP, running inside a Chrome tab.",
   "scripts": {
-    "build": "tsc -b src/tsconfig.json && npm run rollup",
+    "build": "wireit",
+    "build:watch": "npm run tsc:watch && npm run rollup:watch",
```

Comment:
Not sure that this will work as `&&` waits for the first command to finish and exit with code `0` to execute the second one.

Reply:
When we use `&`, the output is garbled. Removed in favor of `wireit`'s `--watch`.